reporter/src/funTest/kotlin/reporters/StaticHtmlReporterTest.kt

```diff
@@ -31,25 +31,27 @@ import java.io.File
 
 import javax.xml.transform.TransformerFactory
 
-class StaticHtmlReporterTest : WordSpec({
-    val ortResult = File("src/funTest/assets/static-html-reporter-test-input.yml")
-            .readValue<OrtResult>()
+class StaticHtmlReporterTest : WordSpec()
+{
```

Comment:
This `{` should be in the line above.
Commit message should start with "StaticHtmlReporterTest: ...".

Reply:
done